Index: pango-1.36.6/pango/modules.c
===================================================================
--- pango-1.36.6.orig/pango/modules.c	2014-08-16 17:49:17.937405157 +0200
+++ pango-1.36.6/pango/modules.c	2014-08-16 17:49:17.933405097 +0200
@@ -582,7 +582,7 @@
 
   if (!file_str)
     {
-      files = g_new (char *, 3);
+      files = g_new (char *, 4);
 
       files[0] = g_build_filename (pango_get_sysconf_subdirectory (),
                                    "pango.modules",
@@ -591,7 +591,11 @@
                                    MODULE_VERSION,
                                    "modules.cache",
                                    NULL);
-      files[2] = NULL;
+      files[2] = g_build_filename (pango_get_lib_subdirectory (),
+                                   MODULE_VERSION,
+                                   "module-files.d",
+                                   NULL);
+      files[3] = NULL;
     }
   else
     files = pango_split_file_list (file_str);
